### DISCUSSION TITLE
How to install previous version of T-Pot

### DISCUSSION THREAD

**Discussion by kriszdargo** [telekom-security/tpotce#1599] [Q&A]
Hi!

I wanted to install the 22.04 version of T-Pot on my host:
git clone --branch 22.04 https://github.com/telekom-security/tpotce.git

But after the install I got the newest 24.04.0 version of the T-Pot and I have problem with the 24.04.0 custom configuration, so I wanted to use the older one.

Can you help me how to install the 22.04 version of T-Pot?

**Comment by t3chn0m4g3** [comment]
Previous versions of T-Pot are no longer supported, you have to use 24.04.

**Reply by kriszdargo** [reply]
Oh, okay. Yesterday, I did a lot with the sensor version before it turns out that it doesn't work.
The problem is that the sensor does not use elasticsearch, but it says e.g. logstash that it needs. I had another problem with the auto creating /data/nginx and the directories in it.
The standard version worked straight away without any problems, so I had to reduce it back with customizer.py.
